src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -134,6 +136,10 @@ const convertFrameworkSummary = (framework) => {
     state: framework.state,
     subState: framework.subState,
     executionType: framework.executionType.toUpperCase(),
+    tags: framework.tags.reduce(
+      (arr, curr) => (arr.includes(curr.name) ? arr : [...arr, curr.name]),
```

Comment:
I notice primary key `uid=hash(frameworkName, tagName)` in the tag table, so we don't need to remove duplicates here.

Reply:
> I notice primary key `uid=hash(frameworkName, tagName)` in the tag table, so we don't need to remove duplicates here.

Removed.